### ISSUE TITLE
onCellsChanged not triggering on fillHandle with Dropdown cell

### ISSUE THREAD

**Issue by telit0** [silevis/reactgrid#403]
I am trying to implement a change when a dropdown cell gets populated with the fillHandle.

**Current behavior**
The onCellsChanged event is not getting triggered

**Expected behavior**
The onCellsChanged event should be triggered the same way as on text cells.

**Your environment details**
 - Device:  desktop
 - OS: macOS/Win11
 - Browser: Edge/Chrome

**Comment by webloopbox** [comment]
I've checked it on the latest version of ReactGrid (4.1.6) and it seems that `onCellsChanged` is triggering correctly. Please take a look at this [sandbox](https://codesandbox.io/p/sandbox/reactgrid-searchable-dropdown-example-r3n5l9?file=%2Fsrc%2Findex.tsx%3A50%2C37&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clwrkmf0q0007356kjvzl7o8x%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clwrkmf0p0003356kneleqf5r%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clwrkmf0p0004356kimhybbha%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clwrkmf0p0006356k2xgtabz2%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clwrkmf0p0003356kneleqf5r%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clwrkmf0p0002356k8bjwam7p%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A50%252C%2522startColumn%2522%253A37%252C%2522endLineNumber%2522%253A50%252C%2522endColumn%2522%253A37%257D%255D%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.tsx%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522id%2522%253A%2522clwrkmf0p0003356kneleqf5r%2522%252C%2522activeTabId%2522%253A%2522clwrkmf0p0002356k8bjwam7p%2522%257D%252C%2522clwrkmf0p0006356k2xgtabz2%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clwrkmf0p0005356kdh7h6hlo%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clwrkmf0p0006356k2xgtabz2%2522%252C%2522activeTabId%2522%253A%2522clwrkmf0p0005356kdh7h6hlo%2522%257D%252C%2522clwrkmf0p0004356kimhybbha%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clwrkmf0p0004356kimhybbha%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A20.91309120987171%257D)

**Comment by telit0** [comment]
Thank you! the example is very helpful with the implementation.

**Comment by testone740** [comment]
@webloopbox Thanks for the example.  Your example has the `currentValue` prop for the dropdown cell, but the documentation does not mention it.  Anyway, I've been trying to implement a cut/paste functionality of selection (similar to the [`Customization` example](https://reactgrid.com/examples/?example=customization)), but have not been able to  figure it out.  Where can I find an example of this feature on Code Sandbox?  Thanks.

Basically, I want to be able to cut a selection value from one cell and paste it into another cell.  When I cut the value, the selection should be blank.
